### PR TITLE
FIX: minor assertion to cover args

### PR DESCRIPTION
**Pull request by kambysese**
@Eric89GXL minor touch up if you're ok with syntax.

### CODE REVIEW

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -949,7 +949,6 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
             if p.trans_to is 'median':
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: crap, not sure how this got in, string comparisons should be `==` not `is`.

Also, why remove this assertion?

Comment (kambysese) [inline mnefun/_mnefun.py]: wdym how it got in? most likely I got it in, you have a check for this or what? 

I rm it bc (i) `trans_to` may also be an `np.array` (ii) thought the `if` statement should be enough, no? I can go with

```
assert isinstance(p.trans_to, (string_types, np.ndarray)) 
             if p.trans_to == 'median':
```
 ?

Comment (Eric89GXL) [inline mnefun/_mnefun.py]: I mean I missed it in review...

Don't worry about putting the assert back. I was initially thinking that
it's bad to keep allowing options that only work with the Python algorithm,
but I think it's okay.

File: mnefun/_mnefun.py [hunk 2]
```diff
@@ -949,7 +949,7 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
+            assert type(p.trans_to) in (str, np.ndarray, type(None))
```
Comment (kambysese) [inline mnefun/_mnefun.py]: @Eric89GXL not sure if this is pretty or acceptable but seems to work.

Comment (Eric89GXL) [inline mnefun/_mnefun.py]: `assert` statements can be skipped by Python when it's executed with `-O`. What you probably want in any case is `isinstance(p.trans_to, (string_types, np.ndarray, type(None)))`

**Comment by kambysese** [comment]
Thanks @Eric89GXL ready for merge...

**Comment by Eric89GXL** [comment]
Thanks @kambysese

**Pull request merged**